### PR TITLE
[TRUNK-5281] Duplicated exceptions in throws clause

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/TRUNK-5281

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ModuleUtil.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ private ModuleUtil() {
 	 *
 	 * @param props Properties (OpenMRS runtime properties)
 	 */
-	public static void startup(Properties props) throws ModuleMustStartException, OpenmrsCoreModuleException {
+	public static void startup(Properties props) throws ModuleMustStartException {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/ModuleUtil.java]: Shouldn't instead be leaving the more specific exception like OpenmrsCoreModuleException?

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa You are right, fixed :)

**Comment by dkayiwa** [comment]
@BartlomiejRasztabiga though this is valid change, i have noticed that it will break existing API clients. Not so?

**Comment by BartlomiejRasztabiga** [comment]
Which for example? 😊

**Comment by dkayiwa** [comment]
This being a public API, you cannot tell because it could be in any module, including proprietary ones. 😊

**Comment by BartlomiejRasztabiga** [comment]
Okay, got it :D Closing

**Pull request closed without merging**